integration-mc1-15/src/main/kotlin/org/eln2/utils/OreGen.kt

```diff
@@ -0,0 +1,71 @@
+package org.eln2.utils
+
+import net.minecraft.block.Block
+import net.minecraft.world.biome.Biome
+import net.minecraft.world.gen.GenerationStage
+import net.minecraft.world.gen.feature.ConfiguredFeature
+import net.minecraft.world.gen.feature.Feature
+import net.minecraft.world.gen.feature.OreFeatureConfig
+import net.minecraft.world.gen.placement.CountRangeConfig
+import net.minecraft.world.gen.placement.Placement
+import net.minecraftforge.registries.ForgeRegistries
+import org.eln2.ModBlocks
+
+/**
+ * Handles ore generation for Eln2
+ */
+class OreGen {
+	companion object {
+		/**
+		 * Configures ore generation in the world. Registers against Forge.
+		 */
+		fun setupOreGeneration() {
```

Comment:
This should definitely be configurable. Add a TODO, at least.

Reply:
I intended to leave this as a function that doesn't need to be called in by the main class, ie, it's a registration method. This is the configurable part; you tell it what blocks, and what levels to put them at. Do you want to clutter the main class with that registration code?

Reply:
It isn't configurable _at the moment_, hence the TODO.

Reply:
What isn't configurable about it exactly? You literally, in one statement, register what ores you want to be generated and how. I don't see how I could make this any more configurable besides adding a literal configuration option in Eln2.json (or whatever our config file will be).

Reply:
Well, yes, that's what I mean. It isn't user-configurable, and it would be easy to forget which things should be when we get around to adding that.

Hence the TODO.

Reply:
Todo added.